witchcraft-server-macros/Cargo.toml

```diff
@@ -14,8 +14,3 @@ proc-macro = true
 proc-macro2 = "1"
 quote = "1"
 syn = { version = "2", features = ["full"] }
-
-[dev-dependencies]
-conjure-error = "4"
-refreshable = "2"
-witchcraft-server = { version = "5.2.0", path = "../witchcraft-server" }
```

Comment:
Drive by fix - new versions of Cargo don't let you publish crates with circular dependencies like this :(